client/src/App.test.js

```diff
@@ -15,13 +17,34 @@ const renderComponent = () =>
     </FetchMock>
   )
 
-test("verifies fetch is called", async () => {
-  const { getByText } = renderComponent()
-  await wait(() => expect(getByText("Hello world")).toBeInTheDocument())
+const renderSingleAppContainer = () =>
+    render(
+        <SingleAppContainer 
+            listNum= {1}
+            url= {"testURL"}
+            title={"Test Title"}
+            category={"test"}
+            rating={4.5} />
+    );
```

Comment:
This should be in its own test file. 

Example
`/Clusters/TopCharts/TopCharts.test.js`
`/Clusters/TopCharts/SingleAppContainer.test.js`